src/utilities/Constants.cpp

```diff
@@ -14,24 +15,21 @@ bool Constants::isInitialized = false;
 bool Constants::robotOutputTargetGrSim = true;
 
 void Constants::init() {
-    //    ros::NodeHandle nh;
-    //   std::string robotOutputTarget;
-    //   nh.getParam("robot_output_target", robotOutputTarget);
-    //   robotOutputTargetGrSim = robotOutputTarget != "serial"; // only use serial if it is explicitly defined
-    std::cout << "robot_output_target = " << (robotOutputTargetGrSim ? "GRSIM" : "SERIAL") << std::endl;
+    std::string target = robotOutputTargetGrSim ? "GRSIM" : "SERIAL";
+    rtt_info("robot_output_target is set to " + target)
     isInitialized = true;
 }
 
 bool Constants::GRSIM() {
     if (!isInitialized) {
-        std::cerr << "Ros::init() was not called yet, but you use a value that depends on a ROS parameter. "
-                  << "\n this may result in unexepected behaviour" << std::endl;
+        rtt_error("You use a value dependent on an unkown environment! This may result in unexepected behaviour")
+        assert(false);
```

Comment:
Why is this added?

Reply:
well if you get this error you have made a serious mistake somewhere, which can be invisible when testing with grsim but can affect real play. This makes sure you crash rightfully.